src/main/java/specifications/BaseSpec.java

```diff
@@ -0,0 +1,35 @@
+package specifications;
+
+import io.restassured.builder.RequestSpecBuilder;
+import io.restassured.builder.ResponseSpecBuilder;
+import io.restassured.http.ContentType;
+import io.restassured.specification.ResponseSpecification;
+
+import java.net.HttpURLConnection;
+
+
+public abstract class BaseSpec {
+
+    public static final String URL = "https://api.trello.com";
```

Comment:
Its not an issue, but BASE_URI / BASE_URL is more relevant naming for this constant